.github/workflows/release.yml

```diff
@@ -2,35 +2,12 @@ name: Release
 
 on:
   push:
-    branches:
-      - main
     tags:
-      - "v*"
-  workflow_dispatch:
+      - "v*.*.*"
```

Comment:
Can we make this more restrictive? IIUC any Supabase member can create their own branch and tag a commit to trigger this.

Reply:
Sure, I've added a rule to protected tags https://github.com/supabase/cli/settings/tag_protection